internal/storage/overlay/conf.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021-2023 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package overlay
+
+import (
+	"time"
+
+	"github.com/cerbos/cerbos/internal/config"
+	"github.com/cerbos/cerbos/internal/storage"
+)
+
+const (
+	confKey                  = storage.ConfKey + ".overlay"
+	defaultFailoverThreshold = 5
+	defaultFailoverInterval  = 5 * time.Minute
+)
+
+// Conf is required (if driver is set to 'overlay') configuration for overlay storage driver.
+// +desc=This section is required only if storage.driver is overlay.
+type Conf struct {
+	// BaseDriver is the default storage driver
+	BaseDriver string `yaml:"baseDriver" conf:"required,example=disk"`
+	// FallbackDriver is the secondary or fallback storage driver
+	FallbackDriver string `yaml:"fallbackDriver" conf:"required,example=git"`
```

Comment:
For the example, I think it might make more sense if the base was `blob` and the failover was `disk`.

```suggestion
	BaseDriver string `yaml:"baseDriver" conf:"required,example=blob"`
	// FallbackDriver is the secondary or fallback storage driver
	FallbackDriver string `yaml:"fallbackDriver" conf:"required,example=disk"`
```